### DISCUSSION TITLE
Next.js App Router Support

### DISCUSSION THREAD

**Discussion by doodirock** [wpengine/faustjs#1482] [Q&A] [answered]
How is the Faust team handling the changes to Nextjs routing?  Are there plans to support App router and/or are there any gotchas we should look out for if starting a new production site with Faust?  Thanks for all the hard work

**Comment by theodesp** [comment, marked as answer]
Hey @doodirock. We have a quarterly goal to provide support for Nextjs App router. We don't have anything concrete now. We will post updates in the Blog for any upcoming changes.

**Reply by doodirock** [reply]
That's awesome news. Looks forward to seeing where Faust goes.
